nautobot/extras/management/commands/runjob.py

```diff
@@ -24,6 +24,11 @@ def add_arguments(self, parser):
             action="store_true",
             help="Commit changes to DB (defaults to dry-run if unset). --username is mandatory if using this argument",
         )
+        parser.add_argument(
+            "--profile",
+            action="store_true",
+            help="Run cProfile on the job execution and write the result to the disk under /tmp (use --local to run the job locally, writing to the local filesystem).",
```

Comment:
The note about `--local` is a bit confusing to me. Isn't it redundant with the help text for `--local` itself?